apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c

```diff
@@ -716,40 +718,36 @@ static void utc_audio_pcm_readi_n(void)
 static void utc_audio_pcm_writei_p(void)
 {
 	int fd;
-	ssize_t size;
 	ssize_t bytes;
 	int ret;
-	unsigned int total_frames_write = 0;
-	unsigned int rate;
-	int play;
+	int total_frames_write = 0;
+	int remain;
 
 	fd = open(AUDIO_TEST_FILE, O_RDONLY);
 	TC_ASSERT_GEQ("pcm_writei", fd, 0);
 
 	/* use default config here */
 	g_pcm = pcm_open(0, 0, PCM_OUT, NULL);
-	TC_ASSERT_GT_CLEANUP("pcm_writei", pcm_is_ready(g_pcm), 0, close(fd));
+	TC_ASSERT_GT("pcm_writei", pcm_get_file_descriptor(g_pcm), 0);
 
-	size = pcm_get_buffer_size(g_pcm);
-	rate = pcm_get_rate(g_pcm);
-	bytes = rate * (pcm_format_to_bits(pcm_get_format(g_pcm)) / 8) * pcm_get_channels(g_pcm) * AUDIO_RECORD_DURATION;
+	bytes = lseek(fd, 0, SEEK_END);
+	lseek(fd, 0, SEEK_SET);
```

Comment:
move lseek above of read to prevent it is more cheap condition for below ASSERT cases.